fitbenchmarking/cli/main.py

```diff
@@ -364,13 +364,19 @@ def main():
     Entry point to be exposed as the `fitbenchmarking` command.
     """
     parser = get_parser()
+    args = parser.parse_args(sys.argv[1:])
 
     if len(sys.argv) == 1:
+        if not args.problem_sets[0].exists():
+            raise FilePathError(
+                "The default problem set has either been "
+                "deleted or moved. Please specify a file "
+                "path to a different problem set with the "
+                "-p option."
+            )
```

Comment:
This is outside of the `exception_handler` which means we get an ugly traceback. Can we exit with a print and `sys.exit`
```suggestion
            print(
                "The default problem set has either been "
                "deleted or moved. Please specify a file "
                "path to a different problem set with the "
                "-p option."
            )
            sys.exit(1)
```

Reply:
I was hoping that eventually we can get rid of all the print statements from the code base. I created an issue for it too. https://github.com/fitbenchmarking/fitbenchmarking/issues/1326. But for the short term, I can see if there is a way to test sys.exit(1). If there is then I don't have an issue with having the print statement here.